iModelCore/ECDb/ECDb/SchemaManagerDispatcher.cpp

```diff
@@ -1627,11 +1628,15 @@ BentleyStatus MainSchemaManager::CanCreateOrUpdateRequiredTables() const
             [co].[IsVirtual] = 0
             AND [cm].[MapStrategy] <> 3
         GROUP BY [cl].[Id], tb.Id
-        HAVING COUNT (*) >= %d;)sql", maxColumns);
+        HAVING COUNT (*) > %d;)sql", maxColumns);

```

Comment:
use SQL parameters and not sprintf() and then use stmt.bindint()

Reply:
Updated code to use SQL parameters and `stmt.BindInt()`